launchdarkly-server-sdk/Cargo.toml

```diff
@@ -25,7 +25,7 @@ lazy_static = "1.4.0"
 log = "0.4.14"
 lru = { version = "0.12.0", default-features = false }
 ring = "0.17.5"
-launchdarkly-server-sdk-evaluation = { git = "https://github.com/launchdarkly/rust-server-sdk-evaluation", branch = "feat/migrations" }
+launchdarkly-server-sdk-evaluation = { git = "https://github.com/launchdarkly/rust-server-sdk-evaluation", branch = "mk/sc-243771/event-sampling" }
```

Comment:
Do we need to make a note to circle back to this, or maybe that is already its own ticket.

Reply:
I have a running list of things I still need to address with this work.